test/crypto/elliptic.js

```diff
@@ -500,10 +499,28 @@ describe('Elliptic Curve Cryptography', function () {
     return Z;
   }
 
+  async function genNodePrivateEphemeralKey(curve, V, d, fingerprint) {
+    const curveObj = new openpgp.crypto.publicKey.elliptic.Curve(curve);
+    const oid = new openpgp.OID(curveObj.oid);
+    const { sharedKey } = await openpgp.crypto.publicKey.elliptic.ecdh.nodePrivateEphemeralKey(
```

Comment:
Could you deduplicate these functions a bit? E.g. by passing 'nodePrivateEphemeralKey' as the first parameter and then doing something like
```suggestion
    const { sharedKey } = await openpgp.crypto.publicKey.elliptic.ecdh[functionName](
```
Then it makes it more clear that they're all doing the same thing